internal/migrations/sql.go

```diff
@@ -371,3 +371,25 @@ func getChangeset(ctx context.Context, db sql.Executor, height int64, index int6
 
 	return changeset, nil
 }
+
+// InGenesisMigration checks if the node is going through a genesis migration.
+// It returns true if the node is still syncing with the old network.
+func InGenesisMigration(ctx context.Context, db sql.ReadTxMaker, endHeight int64) (bool, error) {
+	if endHeight == 0 {
+		return false, nil
+	}
+
+	tx, err := db.BeginReadTx(ctx)
+	if err != nil {
+		return false, err
+	}
+	defer tx.Rollback(ctx)
+
+	// Check if the migration table exists
+	height, err := getLastChangeset(ctx, tx)
+	if err != nil {
+		return false, err
+	}
+
+	return height < endHeight, nil
```

Comment:
What is the interpretation of the start:end range?  Is it half opened like `[start,end)` or is it supposed to include the end?

`<` vs `<=` is based on some convention, but I'm not sure where it's documented.

Reply:
[start height, end height]. I will add documentation to it. 

Also, this code not used anymore, removed this.